### PR TITLE
Adding contentDB note to the mods

### PR DESCRIPTION
**Pull request by Niwla23**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -53,8 +53,8 @@ <h4>Modding API</h4>
     </div>
     <div>
       <h4>Large Collection of Existing Mods</h4>
-      <p>There are over 1,000 open source mods on our forums which are ready to
-         be used, adapted or learned from.
+      <p>There are over 1,000 open source mods on our forums and content.minetest.net which are ready to
```
Comment (Calinou) [inline index.html]:
```suggestion
      <p>There are over 1,000 open source mods on our forums and <a href="https://content.minetest.net">ContentDB</a> which are ready to
```

File: index.html [hunk 2]
```diff
@@ -53,8 +53,8 @@ <h4>Modding API</h4>
...
+         be used, adapted or learned from. You can install them easily in the in-game content store
```
Comment (Calinou) [inline index.html]:
```suggestion
         be used, adapted or learned from. You can install them easily in the in-game content store.
```

**Comment by rubenwardy** [comment]
I purposely avoid the term "content store" in the UI for minetest, as nothing is for sale.

> You can install them easily from within the main menu.

File: index.html (hunk 1 above)
Comment (Niwla23) [reply index.html]: Thats better

**Comment by rubenwardy** [comment]
Also fixed by #215

**Pull request closed without merging**